bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -54,9 +57,12 @@ public static Builder builder() {
   public static class Builder {
     private boolean useTimeout = USE_TIMEOUT_DEFAULT;
     private int shortRpcTimeoutMs = SHORT_TIMEOUT_MS_DEFAULT;
+    private Optional<Integer> shortRpcAttemptTimeoutMs = Optional.absent();
     private int longRpcTimeoutMs = LONG_TIMEOUT_MS_DEFAULT;
     private int mutateRpcTimeoutMs = LONG_TIMEOUT_MS_DEFAULT;
+    private Optional<Integer> mutateRpcAttemptTimeoutMs = Optional.absent();
     private int readRowsRpcTimeoutMs = READ_ROWS_TIMEOUT_MS_DEFAULT;
```

Comment:
FYI - this line was the only merge conflict I dealt with when cherry-picking. On branch, it was "readRowsRpcTimeoutMs = LONG_TIMEOUT_MS_DEFAULT", whereas the value is now READ_ROWS_TIMEOUT_MS_DEFAULT. This did not affect my tests otherwise.